coding/unit_test_snippets/unittest_basic_example.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+
+
+class DemoTestCase1(unittest.TestCase):
+    """Demo test case."""
+
+    def testDemo(self):
+        self.assertTrue(True)
```

Comment:
assertTrue is used far too heavily in our code. I beg you to replace assertTrue and assertFalse in all your examples with a something more generally useful. I suggest assertEqual as the first choice, then throw in something like assertLess (since it suggests the availability of many similar asserts). The other two I suggest you demo are assertAlmostEqual and `with self.assertRaises(...):` Those give a really good overview of the good stuff.